packages/oss-console/src/components/Launch/LaunchForm/LaunchFormComponents/StructInput.tsx

```diff
@@ -99,6 +101,16 @@ export const StructInput: FC<InputProps> = (props) => {
     setParamData(formData);
   }, []);
 
+  useEffect(() => {
+    if (!jsonFormRenderable && initialValue) {
+      const value = initialValue.scalar?.binary?.value;
```

Comment:
instead of having this here, please update `/inputHelpers/utils.ts`->`fromLiteral` to correctly parse the value. Currently that will throw an error